### PR TITLE
fix(Asset): Do not attempt to render images with a status of archived

### PR DESCRIPTION
**Pull request by freakyfelt**
# Purpose of PR

Check the status as part of rendering an `Asset`.
  * If the status is present and is archived short-circuit to render as an asset
  * Else if the type is an image attempt to render a preview using an `img` tag

## PR Checklist

- [x] I have read the relevant `readme.md` file(s)
- [x] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components#commits)
- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Storybook stories are added/updated/not required
- [x] Usage notes are added/updated/not required
- [x] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [x] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/forma-36-react-components/src/components/Asset/Asset.tsx [hunk 1]
```diff
@@ -25,6 +26,7 @@ export type AssetProps = {
   title: string;
   type?: AssetType;
   className?: string;
+  status?: AssetState;
```
Comment (wichniowski) [inline packages/forma-36-react-components/src/components/Asset/Asset.tsx]: I don't think we need an additional prop we can just default to render Asset if no src is present

Comment (wichniowski) [reply packages/forma-36-react-components/src/components/Asset/Asset.tsx]: or even better show a missing image icon

Comment (freakyfelt) [reply packages/forma-36-react-components/src/components/Asset/Asset.tsx]: > I don't think we need an additional prop we can just default to render Asset if no src is present

From yesterday's meeting it sounds like the src is still there, but when you request it the asset may or may not be accessible anymore. This change sets a consistent behaviour in that instance

Comment (freakyfelt) [reply packages/forma-36-react-components/src/components/Asset/Asset.tsx]: Updated the PR title to be clearer

**Pull request merged**